lib/puppet/pops/issues.rb

```diff
@@ -502,7 +502,7 @@ def self.hard_issue(issue_code, *args, &block)
   end
 
   CAPTURES_REST_NOT_SUPPORTED = hard_issue :CAPTURES_REST_NOT_SUPPORTED, :container, :param_name do
-    _("Parameter $%{param} has 'captures rest' - not supported in %{container}") % { param: param_name, container: label.a_an(container) }
+    _("Parameter $%{param} has 'captures rest' which is not supported in an expression of type %{container}.") % { param: param_name, container: label(container) }
```

Comment:
should have a comma after "rest'": (...'captures rest', which...)

Reply:
recommendation accepted.